src/app/containers/ThemeSwitch/index.tsx

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+import { FormLabel } from 'app/components/FormLabel';
+import { Radio } from 'app/components/Radio';
+import styled from 'styled-components/macro';
+import { changeTheme, selectThemeKey } from 'styles/theme/slice';
+import { useDispatch, useSelector } from 'react-redux';
+import { saveTheme } from 'styles/theme/utils';
+import { ThemeKeyType } from 'styles/theme/types';
+
+export function ThemeSwitch() {
+  const theme = useSelector(selectThemeKey);
+  const dispatch = useDispatch();
+
+  const handleThemeChange = (event: React.ChangeEvent<HTMLInputElement>) => {
+    const value = event.target.value as ThemeKeyType;
+    saveTheme(value);
+    dispatch(changeTheme(value));
+  };
+
+  return (
+    <Wrapper>
+      <FormLabel>Select Theme</FormLabel>
+      <Themes>
+        <Radio
+          id="system"
+          label="System theme"
+          className="radio"
+          name="theme"
+          onChange={handleThemeChange}
+          value="system"
+          isSelected={theme === 'system'}
+        />
+        <Radio
+          id="light"
+          label="Light"
+          className="radio"
+          name="theme"
+          onChange={handleThemeChange}
+          value="light"
+          isSelected={theme === 'light'}
+        />
+        <Radio
+          id="dark"
+          label="Dark"
+          className="radio"
+          name="theme"
+          onChange={handleThemeChange}
+          value="dark"
+          isSelected={theme === 'dark'}
+        />
+      </Themes>
+    </Wrapper>
+  );
+}
+
+const Wrapper = styled.div`
```

Comment:
I realize the [airbnb eslint rules](https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/no-multi-comp.md#ignorestateless) technically allow more than one component per file if they are stateless. Is there any preference for whether these components should be moved into another file, or at least above the component that consumes them?